kotest-property/src/commonMain/kotlin/io/kotest/property/arbitrary/codepoints.kt

```diff
@@ -105,6 +105,13 @@ fun Codepoint.Companion.ascii(): Arb<Codepoint> =
       .map { Codepoint(it) }
       .withEdgecases(Codepoint('a'.code))
 
+/**
+ * Returns an [Arb] that generates HEX codepoints.
+ */
+fun Codepoint.Companion.hex(): Arb<Codepoint> =
```

Comment:
We should test this as well, a simple `java.lang.Long.parseLong(hexString, 16)` will work i believe

Reply:
Oh yeah I meant to come back and add tests.
I'll make another PR with tests.